ui/pages/login.tsx

```diff
@@ -102,12 +105,12 @@ const Login: NextPage = () => {
       <div className="p-strip">
         <div className="login-card">
           <div>
-          {flow ?
-          <Card title="Choose Provider" >
-            <Flow onSubmit={onSubmit} flow={flow} />
-          </Card>:
-          <Spinner></Spinner>
-          }
+            {flow ?
+              <Card title="Choose Provider" >
+                <Flow onSubmit={onSubmit} flow={flow} />
+              </Card> :
+              <Spinner></Spinner>
```

Comment:
We don't have any content inside this component, so we can use a self-closing tag here:
```suggestion
              <Spinner />
```